lists/map-correction.tsv

```diff
@@ -0,0 +1,7 @@
+government-domain	hostname	organisation	start-date	end-date
```

Comment:
all these corrections are already in the register?

Reply:
The 'map-corrections.tsv' file got overwritten by the original data.  I've pushed a fix.